data/anaconda.conf

```diff
@@ -101,6 +101,9 @@ gpt = False
 # Tell multipathd to use user friendly names when naming devices during the installation.
 multipath_friendly_names = True
 
+# Do you want to allow imperfect devices?
+allow_imperfect_devices = False
```

Comment:
I would suggest to either define what an imperfect device is (I guess a broken RAID with missing memebers or similar) or link to a place where this is defined.

Reply:
I have added an example.